### PR TITLE
Add Covid Research Paper Connector

### PR DESCRIPTION
**Pull request by madhuparnab**

### CODE REVIEW

**Review by sidhantgoyal-droid — COMMENTED** [review]
the casing is wrong, lets use just "connector" as directory name.
Also should it be at root level, why its under indexer ?

also why we have checked-in research papers ?

File: indexer/covidResearchPaperConnector/covid.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
+{
```
Comment (sidhantgoyal-droid) [inline indexer/covidResearchPaperConnector/covid.json]: we can have a better name for this file:
covid-papers-schema.json

File: indexer/covidResearchPaperConnector/skill-config.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (sidhantgoyal-droid) [inline indexer/covidResearchPaperConnector/skill-config.json]: enrichment-pipeline-config.json ?

File: indexer/covidResearchPaperConnector/src/main/resources/covid_metadata.csv [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cord_uid,title,uri,url,authors
```
Comment (sidhantgoyal-droid) [inline indexer/covidResearchPaperConnector/src/main/resources/covid_metadata.csv]: maybe you can add two files, covid_metadata_small.csv and covid_metadata_large.csv

File: indexer/covidResearchPaperConnector/covid.json (hunk 1 above)
Comment (madhuparnab) [reply indexer/covidResearchPaperConnector/covid.json]: done

File: indexer/covidResearchPaperConnector/skill-config.json (hunk 2 above)
Comment (madhuparnab) [reply indexer/covidResearchPaperConnector/skill-config.json]: done

File: indexer/covidResearchPaperConnector/src/main/resources/covid_metadata.csv (hunk 3 above)
Comment (madhuparnab) [reply indexer/covidResearchPaperConnector/src/main/resources/covid_metadata.csv]: done

**Review by ashish-google — DISMISSED** [review]

**Review by sidhantgoyal-droid — COMMENTED** [review]
Please don't use System outs exception print stacktrace.
Use logger in your classes

**Comment by madhuparnab** [comment]
> Please don't use System outs exception print stacktrace.
> Use logger in your classes

done

**Review by sidhantgoyal-droid — APPROVED** [review]

**Review by ashish-google — APPROVED** [review]

**Pull request merged**